src/Cli.Tests/Snapshots/InitTests.CosmosDbNoSqlDatabase.verified.txt

```diff
@@ -15,7 +15,8 @@
   Runtime: {
     Rest: {
       Enabled: false,
-      Path: /api
+      Path: /api,
+      RequestBodyStrict: true
```

Comment:
Since this is automatically included in Config even for Cosmos, this is why I suggest we not log a warning and instead log an informational event. Warnings may alarm customers and prompt them to attempt to remediate (e.g. have 0 errors and 0 warnings in their code/scripts).

Reply:
Makes sense.

Reply:
Makes sense. We should instead log an info.

Reply:
Per: (https://github.com/Azure/data-api-builder/pull/1608#discussion_r1320482487)

There is actually an actionable thing customers can do: do not specify these flags in the init command. Having a warning may be actually good so that they pay attention to it and know that rest properties would not be honored.

Reply:
agreed. If you confirm this warning does not appear if the option is excluded from the init command, then warning is better than information like you originally had.